x-pack/plugins/apm/server/routes/services/get_services/get_services_items.ts

```diff
@@ -34,6 +36,7 @@ export async function getServicesItems({
   kuery: string;
   setup: ServicesItemsSetup;
   searchAggregatedTransactions: boolean;
+  searchAggregatedServiceMetrics: boolean;
```

Comment:
I'd prefer one option here too. Not a dealbreaker, but I think it's better to make the decision (about what to search) in one place and then pass it on downwards.

Reply:
@dgieselaar I think I agree. Are you suggesting to combine `searchAggregatedTransactions` and `searchAggregatedServiceMetrics` into a single option? In that case I suggest we deprecate `searchAggregatedTransactions` and remove `searchAggregatedServiceMetrics` , and create a new one like:

```ts
transactionSource: 'auto' | 'service-metrics' | 'transaction-metrics' | 'transactions'
```

Reply:
The `auto` in which order should determine the source?

first `'service-metrics'` then `transaction-metrics`?

Reply:
Yes, that order. But we should only use `service-metrics` where it makes sense. Right now only on the inventory page.